yugabyte/src/yugabyte/core.clj

```diff
@@ -103,6 +105,8 @@
          :single-key-acid (with-client single-key-acid/workload (yugabyte.ysql.single-key-acid/->YSQLSingleKeyAcidClient))
          :multi-key-acid  (with-client multi-key-acid/workload (yugabyte.ysql.multi-key-acid/->YSQLMultiKeyAcidClient))
          :append          (with-client append/workload (ysql.append/->Client))
+         :append-rc       (with-client append/workload-rc (ysql.append-rc/->Client))
```

Comment:
Think it makes sense to change our `api/test-name` format to `api/isolation/test-name` (for YSQL at least)